AnimalMatcher/AnimalMatcher.Data/Models/Message.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace AnimalMatcher.Data.Models
+{
+    using AnimalMatcher.Common.Constants;
+    using System.ComponentModel.DataAnnotations;
+
+    public class Message
+    {
+        public int Id { get; set; }
+
+        [Required]
+        public int RecipientId { get; set; }
+
+        public Pet Recipient { get; set; }
+
+        [Required]
+        public int SenderId { get; set; }
+
+        public Pet Sender { get; set; }
+
+        [MaxLength(MessageConstants.SubjectMaxLength)]
+        public string MessageSubject { get; set; }
+        
+        [MaxLength(MessageConstants.BodyMaxLength)]
+        public string MessageBody { get; set; }
```

Comment:
[suggestion] Could be renamed to Body as it is a part of the Message class and it clearly relates to the Message

Reply:
Yep fair call, have corrected this now.